build/Targets/Settings.props

```diff
@@ -178,6 +178,11 @@
     </When>
   </Choose>
 
+  <PropertyGroup>
+    <!-- Don't treat FormattingAnalyzer as an error, even when TreatWarningsAsErrors is specified. -->
+    <WarningsNotAsErrors>$(WarningsNotAsErrors);IDE0055</WarningsNotAsErrors>
```

Comment:
This seems like it will cause IDE055 to warn on every CI leg instead of just the correctness leg.